Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'pry'
 # Prevents bundler from taking a long-time to resolve
 group :development, :test do
   gem 'activerecord-import'
-  gem 'mime-types', '~> 1.16'
+  gem 'mime-types', '>= 1.16', '< 4'
```

Comment:
Probably not. mime-types 3.x is Ruby 2.0+ only. mime-types 2.x is 1.9+ only. mime-types 1.x supports pretty much any version of Ruby, but is no longer receiving any updates at all (it hit EOL on 27 October 2015). When Rails 5 is released, your appraisal gemfile can absolutely restrict to `~> 3.0` because Rails 5 will only support Ruby 2.2+.